src/test/java/de/tum/in/www1/artemis/repository/FeedbackRepositoryTest.java

```diff
@@ -19,23 +21,160 @@ class FeedbackRepositoryTest extends AbstractSpringIntegrationBambooBitbucketJir
     private FeedbackRepository feedbackRepository;
 
     @Test
-    void copyFeedbackWithLongFeedback() {
-        final String longText = "0".repeat(Constants.FEEDBACK_DETAIL_TEXT_DATABASE_MAX_LENGTH + 10);
+    void createFeedbackFromTestCaseCombineMultiple() {
```

Comment:
These tests will be moved with #6802 into `ProgrammingExerciseFeedbackCreationServiceTest.java` which will cause merge conflicts here.